datalad_dataverse/remote.py

```diff
@@ -6,14 +6,17 @@
 from pyDataverse.models import Datafile
 from requests import delete
 from requests.auth import HTTPBasicAuth
+from annexremote import ExportRemote
 from datalad_dataverse.utils import (
     get_native_api,
 )
+import os
+import re
 
 from datalad_dataverse.utils import format_doi
 
 
-class DataverseRemote(SpecialRemote):
+class DataverseRemote(ExportRemote):
```

Comment:
This is now getting the class straight from AnnexRemote, and we are loosing `.message()` with that. Presently no other choice it seems. Just FTR

Reply:
We could inherit from both classes..? That would retain the `.message()` and the ExportRemote functionallity